components/Accordion/Accordion.js

```diff
@@ -45,47 +45,49 @@ function Accordion({ accessibilityId, className, content, hasAnimationOnHover })
   const accordionId = `accordion-control-${accessibilityId}`;
 
   return (
-    <Card
-      className={classNames(styles.Accordion, className)}
-      hasAnimationOnHover={hasAnimationOnHover}
-    >
-      <div className={styles.headingContainer}>{content.headingChildren}</div>
-
-      <button
-        aria-controls={contentId}
-        aria-expanded={isContentVisible}
-        className={styles.accordionToggleButton}
-        data-testid={ACCORDION_TOGGLE_BUTTON}
-        id={accordionId}
-        onClick={toggleAccordionContent}
-        type="button"
-      >
-        {isContentVisible ? (
-          <>
-            <ScreenReaderOnly>{toggleMessages.close}</ScreenReaderOnly>
-            <ChevronDown />
-          </>
-        ) : (
-          <>
-            <ScreenReaderOnly>{toggleMessages.open}</ScreenReaderOnly>
-            <ChevronRight />
-          </>
-        )}
-      </button>
-
-      <section
-        aria-hidden={!isContentVisible}
-        className={classNames(styles.accordionContent, {
-          [styles.visible]: isContentVisible,
-          [styles.invisible]: !isContentVisible,
-        })}
-        data-testid={ACCORDION_CONTENT}
-        id={contentId}
-        style={{ display: isContentVisible ? 'block' : 'none' }}
-      >
-        {content.bodyChildren}
-      </section>
-    </Card>
+    <RadixAccordion.Root type="single" className={classNames(styles.Accordion, className)}>
+      <RadixAccordion.Item value={accordionId}>
+        <RadixAccordion.Header>
+          <RadixAccordion.Trigger
+            className={classNames(styles.headingContainer, {
+              [styles.hover]: hasAnimationOnHover,
+            })}
+            onClick={toggleAccordionContent}
+            aria-controls={contentId}
+            aria-expanded={isContentVisible}
+            data-testid={ACCORDION_TOGGLE_BUTTON}
+            id={accordionId}
+          >
+            <div className={styles.headingContainer}>{content.headingChildren}</div>
+            {isContentVisible ? (
+              <>
+                <ScreenReaderOnly>{toggleMessages.close}</ScreenReaderOnly>
+                <ChevronDown />
+              </>
+            ) : (
+              <>
+                <ScreenReaderOnly>{toggleMessages.open}</ScreenReaderOnly>
+                <ChevronRight />
+              </>
+            )}
+          </RadixAccordion.Trigger>
+        </RadixAccordion.Header>
+        <RadixAccordion.Content>
+          <section
+            aria-hidden={!isContentVisible}
+            className={classNames(styles.accordionContent, {
+              [styles.visible]: isContentVisible,
+              [styles.invisible]: !isContentVisible,
+            })}
+            data-testid={ACCORDION_CONTENT}
+            id={contentId}
+            style={{ display: isContentVisible ? 'block' : 'none' }}
```

Comment:
You'll only want `className` and `data-testid` defined.

The value for `className` will also change quite a bit since you won't need to change visible/invisible styles. Radix will handle that for us.

Reply:
Updated